db/migrate/20140804102641_add_email_to_broadcast_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToBroadcastMessages < ActiveRecord::Migration
+  def change
+    add_column :broadcast_messages, :email, :boolean
```

Comment:
It is used and needed as an indication as to whether a broadcast message has been sent as an email or not - see attached screen shot.
![broadcast_email](https://cloud.githubusercontent.com/assets/1906176/6105979/81efb620-b05d-11e4-86eb-736ed5f37604.png)